raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -2244,7 +2244,6 @@ private Long append(int epoch, List<T> records, boolean isAtomic) {
 
     @Override
     public void close() {
-        log.close();
```

Comment:
Why are we removing this?

Reply:
This has since been done via a separate PR that was already merged.

Reply:
The remove was done here: https://github.com/apache/kafka/pull/10168

Both the `RaftManager` and `KafkaRaftClient` where closing the `ReplicatedLog`. It was decided that the `RaftManager` owns the log and should close it instead of the `KafkaRaftClient`.